Motion/app/build.gradle

```diff
@@ -19,12 +19,12 @@ apply plugin: 'kotlin-android'
 apply plugin: "androidx.navigation.safeargs.kotlin"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
 
     defaultConfig {
         applicationId 'com.example.android.motion'
         minSdkVersion 14
```

Comment:
minSdkVersion is deprecated https://developer.android.com/reference/tools/gradle-api/7.0/com/android/build/api/dsl/BaseFlavor#minSdkVersion(kotlin.Int)

Maybe change this to minSdk ?

Same thing applies for compileSdkVersion and targetSdkVersion

Reply:
Done